config/locales/curation_concerns.en.yml

```diff
@@ -101,17 +102,29 @@ en:
         property: keywords
       title:
         property: name
+
```

Comment:
Iirc that validator is in the WorkBehavior that is included in the Sufia
work behavior for title...
On Apr 5, 2016 6:46 PM, "Chris Beer" <notifications@github.com> wrote:

> In config/locales/curation_concerns.en.yml
> <https://github.com/projecthydra-labs/curation_concerns/pull/716#discussion_r58626737>
> :
>
> > @@ -101,17 +102,29 @@ en:
> >          property: keywords
> >        title:
> >          property: name
> > +
>
> Nope, it was in models, and at least some of it is necessary. It's
> confusing to me what parts in the stack are using it, but I figure
> investigating that is separate from trying to merge the projects.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/projecthydra-labs/curation_concerns/pull/716/files/212f7496bcf388262e5ff46219722043cd319f21#r58626737>
>